<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Load Image</string>
    </property>
    <widget class="QPushButton" name="Loadfolder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Load folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Loadimage_L">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image_L</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Loadimage_R">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>231</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image_R</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>1. Calibration</string>
    </property>
    <widget class="QPushButton" name="Findcorners">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1.1 Find corners</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Findintrinsic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1.2 Find intrinsic</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Calibration</string>
     </property>
     <widget class="QPushButton" name="Findextrinsic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>1.3 Find extrinsic</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="Finddistortion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1.4 Find distortion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Showresult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>1.5 Show result</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>2. Augmented Reality</string>
    </property>
    <widget class="QPushButton" name="Showwordonbroad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 show word on broad</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Showwordvertical">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>2.1 show word vertical</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>3. Stereo disparity map</string>
    </property>
    <widget class="QPushButton" name="stereodisparity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>3.1 stereo disparity map</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>410</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>4. SIFT</string>
    </property>
    <widget class="QPushButton" name="Keypoints">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>4.1 Keypoints</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadImage2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadImage1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MatchedKeypoints">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>4.2 Matched Keypoints</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>5. VGG19</string>
    </property>
    <widget class="QPushButton" name="ModelStructure">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5.2 Show Model Structure</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AugmentedImages">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5.1 Show Augmented Images</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ShowAcc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5.3 Show Acc and Loss</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Inference">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5.4 Inference</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Predict = </string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>221</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Inference Image</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
